src/models/generation/thermal_generation.jl

```diff
@@ -41,7 +41,7 @@ struct TechThermal
     reactivepower::Union{Float64,Nothing} # [MVAr]
     reactivepowerlimits::Union{@NT(min::Float64, max::Float64),Nothing}
     ramplimits::Union{@NT(up::Float64, down::Float64),Nothing}
-    timelimits::Union{@NT(min::Float64, max::Float64),Nothing}
+    timelimits::Union{@NT(up::Float64, down::Float64),Nothing}
```

Comment:
Preference or thoughts between using `up` and `down` versus `upper` and `lower`?